desktop/ui/src/main/java/org/datacleaner/panels/fuse/ColumnListMultipleCoalesceUnitPropertyWidget.java

```diff
@@ -142,17 +145,20 @@ public void removeCoalesceUnit() {
             _unitContainerPanel.updateUI();
             fireBothValuesChanged();
         }
+        updateUI();
```

Comment:
This is a little bit weird call. updateUI() is normally used for a UI (metal, plastic, etc.) change. Don't you want to do a repaint() instead?